### DISCUSSION TITLE
Scheduling Alerts and Checks only during office hours

### DISCUSSION THREAD

**Discussion by ninjamonkey198206** [amidaware/tacticalrmm#767] [Q&A] [answered]
I may have simply overlooked this, as I'm new to the software, but is it possible to set a schedule for all actions for  automation policies, alert policies, and agent check-in alerts?

An example would be having separate policies for servers at a site and workstations, as the servers will likely be up 24/7, while the workstations could only be up Mon - Fri 8am - 5pm.

It could prevent some annoying and unnecessary alerts.

Or is it already baked in somehow, like the agents sending a shutdown notice so the server ignores them until they check in again?

**Comment by sadnub** [comment, marked as answer]
You can apply policies to workstations and servers separately. Right-click on a client or site and go to assign automation policy. As for the alerts, checkout Settings > Alerts Manager. You can create an alert template and in the exclusions (Right-click on the alert template and click Alert Exclusions) you can target only servers.

I recommend only enabling agent overdue alerts on servers and not desktops. Also for any manual server reboots, you can set the agent to maintenance mode, (you can even do this for a whole site by right-clicking on the Site or Client and selecting Enable Maintenance Mode). This will stop all alerts for those agents.

Let me know if this answers your questions. We are always adding more features and improving existing ones, so let me know if there is anything lacking.

Thanks!

**Reply by ninjamonkey198206** [reply]
Ok. I was wondering about how those were handled. I've got various alerts and agent overdue alerts set up on my test PC, which tends to be off for several hours at a time, and once I get used to things I was looking at getting a client set up with it, which would of course require that monthly sub to clear up the A/V hassles.

And I just realized that I haven't had any alerts about the agent being overdue... Balls.

Time to start digging through logs.

Side question:
Is it possible to pay you guys more than one monthly payment without having to use multiple email accounts? (I'm assuming that's how the subscription is handled.)
If I can get clients to go for it, ideally I'd forward the monthly sub onto their bill, per client, to help fund continued development.

**Reply by ninjamonkey198206** [reply]
Nevermind. I just saw that I missed in the documentation that the $50 monthly donation for signed agents was a minimum, so I should be able to bump it up per client that I sign on.
